dcpquery-api.yml

```diff
@@ -148,6 +148,85 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /files/{file_uuid}/bundles:
+      get:
+        operationId: dcpquery.api.files.bundle.get
+        summary: Get list of bundle_fqids for a given file_uuid
+        description: >
+          Given the UUID of a file, returns a list of bundle_fqids belonging to all the bundles the file is a part of.
+        parameters:
+          - name: file_uuid
+            in: path
+            description: Job identifier in RFC4122-compliant UUID format
+            required: true
+            schema:
+              type: string
+              pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+          - name: check_events
+            in: query
+            description: >
+              If set, will check for any recent dss events and check if the file is a part of any unloaded bundles.
```

Comment:
Maybe there's a slightly better word that "unloaded"? Uncommitted? Unindexed?

Reply:
👍